docs/metasploit-framework.wiki/Creating-Your-First-PR.md

```diff
@@ -0,0 +1,122 @@
+# Creating Your First PR - An Intro To Git and the PR Process
+## Intro
+Congratulations fellow traveler, so your interested in contributing to Metasploit eh? Well welcome aboard, its going to be a fun ride!
+You'll learn lots along the way but here are some tips and tricks that should help you get started with making your first PR request
+whilst also avoiding some common pitfalls and learning how some of our systems work.
+
+## Initial Steps and Important Notes
+The rest of this guide assumes you have already followed the steps at [Setting Up A Developer Environment](https://r-7.co/MSF-DEV) in order to get
+a fork of Metasploit set up and ready to run, and that you have added in your SSH keys, set up Ruby and optionally the PostgreSQL database, and
```

Comment:
Can we link to https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account for SSH keys as you did for "Setting Up A Developer Environment"?

Reply:
Sure I'll add that in :)

Reply:
Added in https://github.com/rapid7/metasploit-framework/pull/16949/commits/2bfcac8d8630c70097cd0b029271ec9c25e522d6, let me know if this looks good.